code/game/gamemodes/clock_cult/clock_objects.dm

```diff
@@ -0,0 +1,228 @@
+
+/obj/item/clockwork/slab/attack_self(mob/living/user)
+	if(iscultist(user))
+		user << "<span class='heavy_brass'>\"You reek of blood. You've got a lot of nerve to even look at that slab.\"</span>"
+		user.visible_message("<span class='warning'>A sizzling sound comes from [user]'s hands!</span>", "<span class='userdanger'>[src] suddenly grows extremely hot in your hands!</span>")
+		playsound(get_turf(user), 'sound/weapons/sear.ogg', 50, 1)
+		user.drop_item()
+		user.emote("scream")
+		user.apply_damage(5, BURN, "l_arm")
+		user.apply_damage(5, BURN, "r_arm")
+		return 0
+	if(!is_clockwork_cultist(user))
+		user << "<span class='warning'>The information on [src]'s display shifts rapidly. After a moment, your head begins to pound, and you tear your eyes away.</span>"
+		user.confused = max(0, user.confused + 1)
+		return 0
+	access_display(user)
+
+/obj/item/clockwork/slab/proc/access_display(mob/living/user)
+	if(!is_clockwork_cultist(user))
+		return 0
+	var/action = input(user, "Among the swathes of information, you see...", "[src]") as null|anything in list("Recital", "Records", "Recollection", "Repository")
+	if(!action || !user.canUseTopic(src))
+		return 0
+	switch(action)
+		if("Recital")
+			recite_scripture(user)
+		if("Records")
+			//show_stats(user)
+		if("Recollection")
+			//show_guide(user)
+		if("Repository")
+			show_components(user)
+	return 1
+
+/obj/item/clockwork/slab/proc/recite_scripture(mob/living/user)
+	var/servants = 0
+	var/caches = 0
+	var/unconverted_ai_exists = FALSE
+	for(var/mob/living/M in living_mob_list)
+		if(is_clockwork_cultist(M))
+			servants++
+	for(var/mob/living/silicon/ai/ai in living_mob_list)
+		if(!is_clockwork_cultist(ai))
+			unconverted_ai_exists = TRUE
+	for(var/obj/structure/clockwork/cache in clockwork_caches)
+		caches++
+	var/list/tiers_of_scripture = list("Drivers")
+	tiers_of_scripture += "Scripts[servants >= 5 && caches >= 1 ? "" : " \[LOCKED\]"]"
+	tiers_of_scripture += "Applications[servants >= 8 && caches >= 3 && clockwork_structure_value >= 50 ? "" : " \[LOCKED\]"]"
+	tiers_of_scripture += "Judgement[servants >= 10 && clockwork_structure_value >= 100 && !unconverted_ai_exists ? "" : " \[LOCKED\]"]"
+	var/scripture_tier = input(user, "Choose a category of scripture to recite.", "[src]") as null|anything in tiers_of_scripture
+	if(!scripture_tier || !user.canUseTopic(src))
+		return 0
+	var/list/available_scriptures = list()
+	var/datum/clockwork_scripture/scripture_to_recite
+	var/tier_to_browse
+	switch(scripture_tier)
+		if("Drivers")
+			tier_to_browse = SCRIPTURE_DRIVER
+		if("Scripts")
+			tier_to_browse = SCRIPTURE_SCRIPT
+		if("Applications")
+			tier_to_browse = SCRIPTURE_APPLICATION
+		if("Judgement")
+			tier_to_browse = SCRIPTURE_JUDGEMENT
+	if(!tier_to_browse)
+		user << "<span class='warning'>That section of scripture is too powerful right now!</span>"
+		return 0
+	world << tier_to_browse
+	for(var/datum/clockwork_scripture/S in subtypesof(/datum/clockwork_scripture))
```

Comment:
This line right here - the one that's supposed to look for subtypes - isn't picking anything up. If you look in clock_scriptures.dm, you can even see a dummy subtype I made to try and remedy this. Coderbus had no idea. Any thoughts on why this might be happening?